x/minfee/types/genesis.go

```diff
@@ -0,0 +1,24 @@
+package types
+
+import (
+	"fmt"
+)
+
+// DefaultGenesis returns the default genesis state.
+func DefaultGenesis() *GenesisState {
+	return &GenesisState{
+		NetworkMinGasPrice: DefaultNetworkMinGasPrice, // TODO: remove this field
```

Comment:
I think it will eventually be safe to remove this in a `GenesisStaet` v2, but the current logic should basically ignore this value, and only read from params.